middleware.go

```diff
@@ -196,3 +197,23 @@ func AccessControl(acl Config[ACL], cfg Options) func(next http.Handler) http.Ha
 		})
 	}
 }
+
+// PropagateAccessKey propagates the access key from the context to other webrpc packages.
+func PropagateAccessKey(headerContextFuncs ...func(context.Context, http.Header) (context.Context, error)) func(next http.Handler) http.Handler {
```

Comment:
What are these functions `func(context.Context, http.Header)`?
Is that `func WithHTTPRequestHeaders(ctx context.Context, h http.Header) (context.Context, error)` from Webrpc?

If so, can you please just add comment with that?